src/config/dynamic-theme-fixes.config

```diff
@@ -2112,6 +2130,12 @@ CSS
 .Gor6zc {
     background-color: white !important;
 }
+#cnt #hdtbSum, #cnt>#appbar {
+    background: rgb(24,26,27);
```

Comment:
Please use ${element-default-color} pattern to make fix work in light mode and make it configurable.